Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -1164,7 +1175,7 @@ public override void Tick()
                 def.projectile.soundImpactAnticipate.PlayOneShot(this);
             }
             //TODO : It appears that the final steps in the arc (past ticksToImpact == 0) don't CheckForCollisionBetween.
-            if (ticksToImpact <= 0 || nextPosition.y <= 0f)
+            if ((lerpPosition && ticksToImpact <= 0) || nextPosition.y <= 0f)
```

Comment:
We might need a _different_ flag here, but non-lerp projectiles can have unexpected or unspecified ticksToImpact values, as ticksToImpact isn't really a sensible value when the projectile is launched without a precalculated destination.  Maybe your stuff needs a time fuze type mechanism in play?